editor/editor_settings.cpp

```diff
@@ -959,6 +959,7 @@ void EditorSettings::_load_defaults(Ref<ConfigFile> p_extra_config) {
 	_initial_set("editors/animation/confirm_insert_track", true, true);
 	_initial_set("editors/animation/default_create_bezier_tracks", false, true);
 	_initial_set("editors/animation/default_create_reset_tracks", true, true);
+	_initial_set("editors/animation/insert_at_timeline_cursor", true, true);
```

Comment:
```suggestion
	_initial_set("editors/animation/insert_at_timeline_cursor", false, true);
```

Considering compatibility, the default should be `false`.